gitops/resources/kustomization.go

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2021-2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// From https://github.com/redhat-developer/kam/tree/master/pkg/pipelines/resources
+
+package resources
+
+import "sort"
+
+// Kustomization is a structural representation of the Kustomize file format.
+type Kustomization struct {
+	Resources    []string          `json:"resources,omitempty"`
+	Bases        []string          `json:"bases,omitempty"`
+	CommonLabels map[string]string `json:"commonLabels,omitempty"`
+}
```

Comment:
does it need the apiVersion and kind?

<img width="807" alt="Screen Shot 2022-01-20 at 4 08 31 PM" src="https://user-images.githubusercontent.com/31771087/150422494-4444ddd4-ca16-4b49-95d1-a53874e7a802.png">

Reply:
It's straight from kam's library, and doesn't seem to be affect kustomize at all, so I'd probably leave it as-is for now.